hydra-access-controls/lib/hydra/config.rb

```diff
@@ -11,6 +11,8 @@ def []= key, value
           self.permissions = value
         when :user_model
           self.user_model = value
+        when :id_to_resource_uri
```

Comment:
The answer is no we cannot. This is for converting id to the externally visibile id e.g. `http://oregondigital.org/resources/:id`